.github/workflows/ci.yml

```diff
@@ -61,11 +56,26 @@ jobs:
           truncate_stack_traces: false
 
   reporting-jacoco-coverage-failures:
-    name: "report-jacoco-coverage-failures"
+    name: "coverage-jacoco"
     needs: run-unit-tests
     uses: ./.github/workflows/worker.yml
     if: ${{ !contains( github.event.pull_request.labels.*.name, 'jacoco:skip') }}
     with:
       script: .github/scripts/create-jacoco-coverage-report.sh
       artifacts_to_download: "unit-test-reports-*"
-      key: "jacoco-coverage-report"
+      key: "coverage-jacoco"
+
+  run-unit-tests-all-jdk:
+    needs: test-report
+    name: "unit tests(all jdk)"
+    if: ${{ !cancelled() && ( contains( github.event.pull_request.labels.*.name, 'tests:all-jdk') || github.event_name == 'push' ) }}
+    strategy:
+      fail-fast: false
+      matrix:
+        pattern: [ "A*,G*,R*", "B*,O*,S*,X*,Y*,Z*", "C*,E*", "D*,J*,K*", "F*,H*,U*", "I*,N*,T*", "L*,Q*,W*", "M*,P*,V*"]
+        jdk: [ "11", "21.0.4" ]
```

Comment:
Small nit: Currently, in the UI, it shows up as:
1. AGR (11)
2. AGR (21.0.4)
3. BOSXYZ (11)
4. BOSXYZ (21.0.4)
5. and so on

I think if we switch the order to:
```
        jdk: [ "11", "21.0.4" ]
        pattern: [ "A*,G*,R*", "B*,O*,S*,X*,Y*,Z*", "C*,E*", "D*,J*,K*", "F*,H*,U*", "I*,N*,T*", "L*,Q*,W*", "M*,P*,V*"]
```

Then it will show up like:
1. AGR (11)
2. BOSXYZ (11)
3. other (11) ones
4. AGR (21.0.4)
5. BOSXYZ (21.0.4)
6. and so on

This seems better to me from an easy-to-look-and-find point of view. It will also make it consistent with the groups listed for jdk17.